pipelines/README.md

```diff
@@ -1,8 +1,7 @@
-# This is not a good place for pipelines
+# Temporary DECam Pipelines
```

Comment:
I suppose I'll allow it 😜

Reply:
I forgot I made this edit! Seemed like a good idea... but sad to see the old message go!